### ISSUE TITLE
Xiaomi device is in supported list but my log says it isn't, this let the software crash and every sensor stop reporting status

### ISSUE THREAD

**Issue by Nihvel** [Koenkk/zigbee-shepherd-converters#521]
I have 2x Xiaomi Power Plug: `ZNCZ02LM`
row 367: https://github.com/Koenkk/zigbee-shepherd-converters/blob/master/devices.js

My log, during pairing, write:
```
'{"version":"1.4.0","commit":"unknown","coordinator":20190223,"log_level":"info","permit_join":true}'
  zigbee2mqtt:error 7/15/2019, 7:27:47 PM Cannot get the Node Descriptor of the Device: 0x00158d000315ae37 (Error: Timed out after 10000 ms)
  zigbee2mqtt:info 7/15/2019, 7:27:48 PM **New device 'undefined' with address 0x00158d000315ae37 connected**!
  zigbee2mqtt:info 7/15/2019, 7:27:48 PM MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_connected","message":"0x00158d000315ae37","meta":{}}'
  zigbee2mqtt:warn 7/15/2019, 7:27:48 PM **Device with modelID 'undefined' is not supported**.
  zigbee2mqtt:warn 7/15/2019, 7:27:48 PM Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html
  zigbee2mqtt:info 7/15/2019, 7:27:48 PM MQTT publish: topic 'homeassistant/switch/0x00158d000315ae37/switch/config', payload '{"payload_off":"OFF","payload_on":"ON","value_template":"{{ value_json.state }}","command_topic":"zigbee2mqtt/0x00158d000315ae37/set","state_topic":"zigbee2mqtt/0x00158d000315ae37","name":"0x00158d000315ae37_switch","unique_id":"0x00158d000315ae37_switch_zigbee2mqtt","device":{"identifiers":"zigbee2mqtt_0x00158d000315ae37","name":"0x00158d000315ae37","sw_version":"Zigbee2mqtt 1.4.0","**model":"Mi power plug ZigBee (ZNCZ02LM)**","manufacturer":"Xiaomi"},"availability_topic":"zigbee2mqtt/bridge/state"}'
  zigbee2mqtt:info 7/15/2019, 7:27:48 PM MQTT publish: topic 'homeassistant/sensor/0x00158d000315ae37/power/config', payload '{"unit_of_measurement":"Watt","icon":"mdi:flash","value_template":"{{ value_json.power }}","state_topic":"zigbee2mqtt/0x00158d000315ae37","json_attributes_topic":"zigbee2mqtt/0x00158d000315ae37","name":"0x00158d000315ae37_power","unique_id":"0x00158d000315ae37_power_zigbee2mqtt","device":{"identifiers":"zigbee2mqtt_0x00158d000315ae37","name":"0x00158d000315ae37","sw_version":"Zigbee2mqtt 1.4.0","model":"Mi power plug ZigBee (ZNCZ02LM)","manufacturer":"Xiaomi"},"availability_topic":"zigbee2mqtt/bridge/state"}'
  zigbee2mqtt:info 7/15/2019, 7:27:48 PM MQTT publish: topic 'homeassistant/sensor/0x00158d000315ae37/linkquality/config', payload '{"unit_of_measurement":"-","value_template":"{{ value_json.linkquality }}","state_topic":"zigbee2mqtt/0x00158d000315ae37","json_attributes_topic":"zigbee2mqtt/0x00158d000315ae37","name":"0x00158d000315ae37_linkquality","unique_id":"0x00158d000315ae37_linkquality_zigbee2mqtt","device":{"identifiers":"zigbee2mqtt_0x00158d000315ae37","name":"0x00158d000315ae37","sw_version":"Zigbee2mqtt 1.4.0","model":"Mi power plug ZigBee (ZNCZ02LM)","manufacturer":"Xiaomi"},"availability_topic":"zigbee2mqtt/bridge/state"}'
  zigbee2mqtt:info 7/15/2019, 7:27:49 PM Zigbee publish to device '0x00158d000315ae37', genTime - readRsp - [{"attrId":0,"status":0,"attrData":616530470,"dataType":226}] - {"direction":1,"seqNum":16,"disDefaultRsp":1} - 1
  zigbee2mqtt:warn 7/15/2019, 7:27:58 PM Message without device!
```
..And the device fails to pair. 

**The other device** I have, where I tried for days to have it working, prints the same log about it not being supported.
Eventually it got integrated in HA but messes up the configuration so bad that every sensor stop working after some minutes. Repeatedly. 
I notice something is wrong when the log start writing:
` zigbee2mqtt:error 7/9/2019, 8:48:00 PM Failed to ping 0x00158d0002fc073e`

Also, both these power plugs are seen as router in `database.db` file (where the other sensors are EndDevice):
`{"id":8,"type":"Router","ieeeAddr":"0x00158d0002fc073e","nwkAddr":14030,"manufId":4447,"epList":[1,2,3],"status":"online","joinTime":1563210123,"endpoints":{"1":{"profId":260,"epId":1,"devId":81,"inClusterList":[0,1,2,3,4,5,6,10,16],"outClusterList":[10,25],"clusters":{}},"2":{"profId":260,"epId":2,"devId":9,"inClusterList":[12],"outClusterList":[4,12],"clusters":{}},"3":{"profId":260,"epId":3,"devId":83,"inClusterList":[12],"outClusterList":[12],"clusters":{}}},"_id":"wIb7i99QfGltLMAM"}`

The database file contains many other rows for `0x00158d0002fc073e`, they all got created when the coordinator was in pairing mode and this power plug kept on sending mqtt messages. All of these rows, in fact, have different and several values for the last value: `"_id":"wIb7i99QfGltLMAM"}`

Now I removed these 2 plugs from database and state file. Both using the mqtt message for removing device(s) and manually by removing their row from above file.
The zigbee network works great when these plugs aren't integrated. 

There's something wrong and I will be available for submitting more or detailed info if this helps troubleshooting the problem.
Thanks for this great work!

**Comment by Nihvel** [comment]
I'm currently running on this firmware:
`zigbee2mqtt:info 7/16/2019, 10:34:50 AM Coordinator firmware version: '20190223'`
and I noticed an updated one is available.

I will upgrade the sniffer and try again pairing the device(s).

**Comment by Nihvel** [comment]
No, nothing changed using the latest firmware.
```
  zigbee2mqtt:info 7/17/2019, 12:48:20 AM Device incoming...
  zigbee2mqtt:info 7/17/2019, 12:48:20 AM MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"pairing","message":"device incoming"}'
  zigbee2mqtt:warn 7/17/2019, 12:48:26 AM Message without device!
  zigbee2mqtt:warn 7/17/2019, 12:48:33 AM Device with modelID 'undefined' is not supported.
  zigbee2mqtt:warn 7/17/2019, 12:48:33 AM Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html
  zigbee2mqtt:info 7/17/2019, 12:48:34 AM MQTT publish: topic 'zigbee2mqtt/0x00158d0002fc073e', payload '{"state":"ON","power":0,"voltage":null,"consumption":1.03,"temperature":25,"linkquality":81}'
```

In some minutes or less than a hour it will probably start printing `Failed to ping`  and the whole zigbee network will crash. I will update this comment so to let you sir know what happened after this latest tentative of pairing.
It's night time here, I will most probably update the comment in some hours.
Thanks again!

**Comment by Nihvel** [comment]
I do not want to yell "Victory" so quickly: After i "almost" paired the device '073e' and not getting any error or crash, I played around with it and after 30 minutes it seems it is pretty stable.

I paired the other device, 'ae37'. This, didn't print the error about "Device with modelID undefined". It didn't. Strange but great.
It got paired. Here as well, after some minutes, the zigbee network is still up and the service is up and running as well.

What if the firmware upgrade, indeed, fixed it? Again, I do not want to say it's over so quick. I've been struggling to set up these 2 devices for weeks. I will wait until tomorrow, if something happens it is within the first hours, just like the first time i noticed something was not right.
